Objects/dictobject.c

```diff
@@ -3108,10 +3123,17 @@ _PyDict_MergeEx(PyObject *a, PyObject *b, int override)
     return dict_merge(interp, a, b, override);
 }
 
+/*[clinic input]
+dict.copy
+
+Returns a shallow copy of D.
```

Comment:
What is `D`?
"Returns a shallow copy." or "Returns a shallow copy of the dict." would be better.